docs/ipfs/release-flow.md

```diff
@@ -0,0 +1,54 @@
+# Release Flow
+
+Releasing the widget to IPFS involves several steps, including:
+1. Creating a content-addressable archive (CAR file) containing the necessary files for the widget to function.
+2. Uploading the CAR file to the IPFS network using an IPFS provider.
+3. Pinning the uploaded CAR file to ensure its permanent availability on the IPFS network.
+
+:::note IPFS Pinning
+By default, IPFS nodes only keep data in their cache for a limited time.
+Then, the data is removed by an automatic garbage collection process.
+To ensure that content remains available on the IPFS network permanently,
+it must be pinned using its Content Identifier (CID).
+:::
+
+To complete and automate these and other required steps, we leverage the power of GitHub Actions.
+
+## GitHub Actions workflow
+
+We've already been using GitHub Actions to build and deploy our applications,
```

Comment:
GitHub Actions has already been used to build and deploy applications, that’s why it is used to adapt them for IPFS releases as well.

Reply:
+1